examples/processors/parse_sequentially.tf

```diff
@@ -84,3 +84,11 @@ resource "mezmo_http_destination" "destination3" {
   uri         = "https://example.org"
   inputs      = [mezmo_parse_sequentially_processor.processor1.parsers.2.output_name]
 }
+
+resource "mezmo_blackhole_destination" "destination4" {
+  pipeline_id = mezmo_pipeline.pipeline1.id
+  title       = "Http desintation"
+  description = "Send unmatched data to a blackhole"
+  uri         = "https://example.org"
+  inputs      = [mezmo_parse_sequentially_processor.processor1.unmatched]
```

Comment:
I think the API expects `<component_id>.<output_name>` to identify the input. For example, `2994040b-cff7-4138-8d8c-6c43ec5914d2._unmatched`.

If that's correct we should either change `unmatched` and all the `output_name` instances to return `<component_id>.<output_name>` or we should suggest the user to enter:

```tf
inputs = ["${mezmo_parse_sequentially_processor.processor1.id}.${mezmo_parse_sequentially_processor.processor1.unmatched}"]
```